core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraSession.kt

```diff
@@ -878,6 +902,20 @@ private fun Preview.Builder.updateCameraStateWithCaptureResults(
                 result: TotalCaptureResult
             ) {
                 super.onCaptureCompleted(session, request, result)
+
+                if (BuildCompat.isAtLeastV()) {
```

Comment:
Since we're no longer using the preview SDK, this should be changed to:
`Build.VERSION.SDK_INT >= Build.VERSION_CODES.VANILLA_ICE_CREAM`